docs/PUBLISHING.md

```
@@ -0,0 +1,557 @@
+# マイナーバージョンを上げて公開
+vsce publish minor
+
+# メジャーバージョンを上げて公開
+vsce publish major
+```
+
+### 2. 公開の確認
+
+公開後、以下で確認できます：
+
+- **Marketplace**: https://marketplace.visualstudio.com/items?itemName=<publisher>.<name>
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)